include/tvm/tir/stmt.h

```diff
@@ -1316,6 +1316,14 @@ constexpr const char* fragment_layout = "fragment_layout";
  * \brief Mark that the kernel is hand threaded and doesn't need syncs inserted
  */
 constexpr const char* hand_threaded = "hand_threaded";
+
+/*!
+ * \brief Mark that whether a block need to be complete access region during script parsing.
```

Comment:
```suggestion
 * \brief Mark whether a block need to be complete access region during script parsing.
```

Can you clarify this part?  `to be complete access region` this doesn't parse in English for me.

Reply:
Mark whether the auto-completer need to fill in missing access region during script parsing.